ph5/utilities/tests/test_obspytoph5.py

```diff
@@ -1,47 +1,61 @@
 '''
-Tests for metadatatoph5
+Tests for obspytoph5
 '''
-import unittest
-from ph5.utilities import obspytoph5
-from ph5.utilities import initialize_ph5
 import os
-from ph5.utilities import metadatatoph5
 import sys
+import shutil
+import unittest
+
 from mock import patch
-from ph5.core.tests.test_base import initialize_ex
+from testfixtures import OutputCapture
 
+from ph5.utilities import obspytoph5
+from ph5.utilities import metadatatoph5
+from ph5.core.tests.test_base import LogTestCase, TempDirTestCase,\
+    initialize_ex
 
-class TestObspytoPH5(unittest.TestCase):
 
+class TestObspytoPH5(TempDirTestCase, LogTestCase):
     def setUp(self):
-        self.path = 'ph5/test_data/miniseedph5'
+        super(TestObspytoPH5, self).setUp()
+        self.station_xml_path = os.path.join(
+            self.home, 'ph5/test_data/metadata/station.xml')
+        # copy miniseed to tmpdir
+        miniseed_dir = os.path.join(self.home, 'ph5/test_data/miniseed')
+        for filename in os.listdir(miniseed_dir):
+            file_path = os.path.join(miniseed_dir, filename)
+            shutil.copy(file_path, '.')
+
+        if 'test_main' not in self._testMethodName:
```

Comment:
These test_mains should be moved to another class

Reply:
will move test_mains to another class